src/test/java/com/stripe/functional/TelemetryTest.java

```diff
@@ -26,7 +28,7 @@
 public class TelemetryTest extends BaseStripeTest {
   @Test
   public void testTelemetryEnabled() throws StripeException, IOException, InterruptedException {
-    MockWebServer server = new MockWebServer();
+    @Cleanup MockWebServer server = new MockWebServer();
```

Comment:
ah I learned something new!
I suppose we never "close" this webserver resource in the first place?

Reply:
Yes, previously the server was "leaked" because we never called `close()`. Lombok's [`@Cleanup`](https://projectlombok.org/features/Cleanup) annotation is super useful to deal with such resources and we already use it in a few other places.